MonsterTrainModdingAPI/Builders/CharacterDataBuilder.cs

```diff
@@ -54,6 +54,14 @@ public class CharacterDataBuilder
         public string AssetPath { get; set; }
         public AssetBundleLoadingInfo BundleLoadingInfo { get; set; }
```

Comment:
This appears to be unused. Did you forget to remove it after adding the two more specific ones below it?